submissions/Harnytskyi/html_css_popup/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <link rel="stylesheet" href="css/style.css">  
+        <link rel="icon" href="img/favicon.png">
+        <title>Popup</title>
+    </head>
+    <body>
+        <nav>
+            <ul class="menu">
+                <li class="menu-item">
+                    <a href="#">Mail</a>
+                </li>
+                <li class="menu-item">
+                    <a href="#">Images</a>
+                </li>
+                <li class="menu-item">
+                    <input class="popup-check" id="popup-check" type="checkbox" />
+                        <label for="popup-check">
+                            <img alt="menu-button" class="menu-button-img" src="./img/popup-button.png">
+                        </label>
+                    
+
+                    <div class="popup">
```

Comment:
It would be better to use tags `nav > ul > li` for menu-icons in the popup. Could you please add these tags